docs/examples/drupal7/index.md

```diff
@@ -213,42 +213,40 @@ This Makefile pulls everything above together into a single file. It takes
 advantage of some code reuse, and cleans up temp files at the end to keep the
 preview disk space usage down a little.
 
-**Make sure the indents are TABs if this is copied & pasted.**
-
-```sh
+<pre><code class="lang-sh">
 packages:
```

Comment:
This should be on the same line as `<pre><code>` to avoid extra whitespace at the top of the rendered code block

Reply:
I mentioned that in Slack, but I'm guessing you missed it. I think it's easier to read the source as it is. The extra white-space doesn't hurt anything and it makes it easier to maintain as is IMO. 

https://lullabot.slack.com/archives/C0PHQ395K/p1522163770000111

Reply:
I feel like the maintenance difference would probably be negligible (minimal, at least), and would rather the output of all of the code blocks be consistent throughout the doc.